common/include/pcl/impl/point_types.hpp

```diff
@@ -506,20 +506,21 @@ namespace pcl
   PCL_EXPORTS std::ostream& operator << (std::ostream& os, const PointXYZRGBA& p);
   /** \brief A point structure representing Euclidean xyz coordinates, and the RGBA color.
     *
-    * The RGBA information is available either as separate r, g, b, or as a
-    * packed std::uint32_t rgba value. To pack it, use:
+    * The RGBA information is available either as separate r, g, b and a uint8_t values,
+    * or as a packed std::uint32_t rgba value. To pack it, use:
     *
     * \code
-    * int rgb = ((int)r) << 16 | ((int)g) << 8 | ((int)b);
+    * int rgba = ((int)a) << 24 | (int)r) << 16 | ((int)g) << 8 | ((int)b);
```

Comment:
```suggestion
    * std::uint32_t rgba = ((std::uint32_t)a) << 24 | ((std::uint32_t)r) << 16 | ((std::uint32_t)g) << 8 | ((std::uint32_t)b);
```
Please also change to `std::uint32_t`, as `int` may give unexpected results when shifting.